src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,52 @@
+import { Loader } from '../Loader';
+import { PeopleTable } from '../Loader/PeopleTable';
+import { getPeople } from '../services/api';
```

Comment:
Code organization: It's better to separate local and third-party imports. All third-party imports should go first, then local imports. This will make the code easier to read and maintain.